src/main/java/ti4/helpers/async/RoundSummaryHelper.java

```diff
@@ -66,21 +68,31 @@ public static void finishEditRoundSummary(ModalInteractionEvent event, Game game
         if (matcher.matches()) {
             ModalMapping mapping = event.getValue("summary");
             String thoughts = mapping.getAsString();
-            storeEndOfRoundSummary(game, player, matcher.group("round"), thoughts, false);
+            storeEndOfRoundSummary(game, player, matcher.group("round"), thoughts, false, event.getChannel());
         }
     }
 
-    public static void storeEndOfRoundSummary(Game game, Player player, String roundNum, String thoughts, boolean append) {
+    public static void storeEndOfRoundSummary(Game game, Player player, String roundNum, String thoughts, boolean append, MessageChannel eventChannel) {
         roundNum = roundNum.replaceAll("[^0-9]", ""); // I only want the digits
-        String roundKey = "endofround" + roundNum + player.getFaction();
+        String roundKey = resolveRoundSummaryKey(player, roundNum);
         String previousThoughts = "";
         if (append && !game.getStoredValue(roundKey).isEmpty()) {
             previousThoughts = game.getStoredValue(roundKey);
             previousThoughts = previousThoughts.replaceFirst("\\.? ?$", "") + "\n";
         }
         game.setStoredValue(roundKey, previousThoughts + thoughts);
 
-        MessageHelper.sendMessageToChannelWithButton(player.getCardsInfoThread(), player.getFactionEmoji() + " stored an end of round summary", Buttons.EDIT_SUMMARIES);
+        MessageChannel playerChannel = player.isRealPlayer() ? player.getCardsInfoThread() : eventChannel;
```

Comment:
you can check, but player.getCardsInfoThread() _should_ still work for any not-real player

Reply:
Yes getCardsInfoThread will create a thread if one would not exist. Didn't want card thread to be made for GM in FoW. They have their GM channel (which we have no good way of getting other than searching by channel name) and this seemed the simplest way to get endofround summaries to work from there.